BIPs/BIP-195.json

```diff
@@ -0,0 +1,65 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1677103514316,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.13.2",
+    "createdFromSafeAddress": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f",
+    "createdFromOwnerAddress": "",
+    "checksum": "0xfd22559117dce8496e5757f7bd8e97870d19c3f05d72603e901c5ce8c682d9c5"
+  },
+  "transactions": [
+    {
+      "to": "0x8F42aDBbA1B16EaAE3BB5754915E0D06059aDd75",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          {
+            "internalType": "address",
+            "name": "target",
+            "type": "address"
+          },
+          {
+            "internalType": "bytes",
+            "name": "data",
+            "type": "bytes"
+          }
+        ],
+        "name": "performAction",
+        "payable": true
+      },
+      "contractInputsValues": {
+        "target": "0xFBf87D2C22d1d298298ab5b0Ec957583a2731d15",
+        "data": "0xab8f0945"
+      }
+    },
+    {
+      "to": "0x8F42aDBbA1B16EaAE3BB5754915E0D06059aDd75",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          {
+            "internalType": "address",
+            "name": "target",
+            "type": "address"
+          },
+          {
+            "internalType": "bytes",
+            "name": "data",
+            "type": "bytes"
+          }
+        ],
+        "name": "performAction",
+        "payable": true
+      },
+      "contractInputsValues": {
+        "target": "0xF0ea3559Cf098455921d74173dA83fF2f6979495",
```

Comment:
search for "MAI" on veBAL voting page and you'll find an arbitrum pool with mai/usdc/usdt which is this pool. naming conventions weren't great back then heh.

it's true you can't verify pool ID but you can verify the beginning of the pool ID in the URL matches that contract address. pool ID's always start with the [pool contract address](https://arbiscan.io/address/0x0510ccf9eb3ab03c1508d3b9769e8ee2cfd6fdcf). 

Also, we do not kill L2 gauges since that serves no real purpose. we kill the root gauge on Ethereum - that prevents any further minting of BAL to the gauge. L2 gauge then stops receiving BAL by default.